src/OpenApi/Generate.elm

```diff
@@ -698,14 +701,56 @@ toRequestFunctions server effectTypes method pathUrl operation =
+                    SchemaUtils.typeToEncoder True type_
+                        |> CliMonad.map
+                            (\encoder config ->
+                                let
+                                    encoded : Elm.Expression
+                                    encoded =
+                                        Elm.get "body" config
+                                            |> encoder
+                                            |> Gen.Json.Decode.decodeValue (Gen.Json.Decode.dict Gen.Json.Decode.value)
+                                            |> Gen.Result.map
+                                                (\keyValues ->
+                                                    keyValues
+                                                        |> Gen.Dict.toList
+                                                        |> Gen.List.call_.map
+                                                            (Elm.fn ( "keyVal", Nothing )
+                                                                (\keyVal ->
+                                                                    Elm.Let.letIn
+                                                                        (\( key, value ) ->
+                                                                            Gen.String.call_.concat
+                                                                                (Elm.list
+                                                                                    [ Gen.Url.call_.percentEncode key
+                                                                                    , Elm.string "="
+                                                                                    , Gen.Url.call_.percentEncode (Gen.Json.Encode.encode 0 value)
+                                                                                    ]
+                                                                                )
+                                                                        )
+                                                                        |> Elm.Let.tuple "key" "value" keyVal
+                                                                        |> Elm.Let.toExpression
+                                                                )
+                                                            )
+                                                        |> Gen.String.call_.join (Elm.string "&")
+                                                )
+                                            |> Gen.Result.withDefault (Elm.string "")
+                                in
+                                { core = Gen.Http.call_.stringBody (Elm.string "application/x-www-form-urlencoded") encoded
+                                , elmPages = Gen.BackendTask.Http.jsonBody encoded
+                                , lamderaProgramTest = Gen.Effect.Http.jsonBody encoded
+                                }
```

Comment:
Why the difference between the three?

Reply:
🤦 that I forgot to update the other 2. Hooray for code reviews!